scripts/helper.js

```diff
@@ -761,6 +761,30 @@ var permissions = [{
 },{
   target: `${accounts.settings.account}@referendum`,
   action: 'conffloat'
+},{
+  target: `${accounts.campaignbank.account}@active`,
+  actor: `${accounts.dao.account}@eosio.code`
+}, {
+  target: `${accounts.milestonebank.account}@active`,
+  actor: `${accounts.dao.account}@eosio.code`
+}, {
+  target: `${accounts.alliancesbank.account}@active`,
+  actor: `${accounts.dao.account}@eosio.code`
+},{
+  target: `${accounts.bank.account}@active`,
+  actor: `${accounts.dao.account}@eosio.code`
+},{
+  target: `${accounts.escrow.account}@active`,
+  actor: `${accounts.dao.account}@eosio.code`
+},{
+  target: `${accounts.dao.account}@active`,
+  actor: `${accounts.escrow.account}@eosio.code`
+},{
+  target: `${accounts.dao.account}@active`,
+  actor: `${accounts.onboarding.account}@eosio.code`
+},{
+  target: `${accounts.onboarding.account}@active`,
+  actor: `${accounts.dao.account}@eosio.code`
```

Comment:
@IanMendozaJaimes 

Dao basically has the same rights as referendums + proposals combined right